src/shared/local/moveDetection.ts

```diff
@@ -73,7 +81,14 @@ export const getMatches = (status: StatusRow[]): AddedAndDeletedFilenames => {
   return { added: addedFilenamesWithMatches, deleted: deletedFilenamesWithMatches };
 };
 
-/** build maps of the add/deletes with filenames, returning the matches  Logs if non-matches */
+export const getLogMessage = (matches: StringMapsForMatches): string =>
```

Comment:
with the more complex result object, this made more sense to handle outside the ShadowRepo class